frontend/src/views/main/project/TestsCatalog.vue

```diff
@@ -197,12 +199,44 @@ const selectedTestUsage = computed(() => {
       .map('type')
       .uniq()
       .value()
-      .filter(i => i !== 'str'),
-    selected.value.name
+      .filter(i => i !== 'str')
```

Comment:
I think that we should ignore any primitives (`int`, `float`, '`str`, `bool`, ...).

Reply:
Furthermore we also only need to import when the default value is set

Reply:
> Furthermore we also only need to import when the default value is set

The line 196 creates an array of only required params